01week/rockPaperScissors.js

```diff
@@ -9,9 +9,59 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+	// get rid of whitespace and convert to lower case before beginning to check
+	hand1 = hand1.trim().toLowerCase();
+	hand2 = hand2.trim().toLowerCase();
 
-  // Write code here
+	// only accept rock, paper, or scissors
+	if((hand1 === 'rock' || hand1 === 'paper' || hand1 === 'scissors') &&
```

Comment:
You repeat yourself in this if statement, what would be a more efficient way to check if you had a valid entry?